src/sentry/integrations/pagerduty/client.py

```diff
@@ -27,16 +36,18 @@ def send_trigger(self, data):
         if isinstance(data, EventCommon):
             source = data.transaction or data.culprit or "<unknown>"
             group = data.group
+            level = data.get_tag("level") or "error"
+            custom_details = serialize(data, None, ExternalEventSerializer())
             payload = {
```

Comment:
Feel like it would be worth it to put down the link for the API format for anyone seeing it in the future: https://v2.developer.pagerduty.com/docs/send-an-event-events-api-v2